get-fedora-latest-config

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+if [ "${VERBOSE:-0}" -ge 2 ] || [ "${DEBUG:-0}" -eq 1 ]; then
+    set -x
+fi
+
+localdir="$(dirname "$(readlink -f "$0")")"
+releasever="$1"
+
+kernelver="$(cat "$localdir/version")"
+kernelsrc="linux-$kernelver"
+
+[[ "x$releasever" == "x" ]] && { echo "Please provide Fedora release version as first argument, e.g. '29'."; exit 1; }
+
+# baseurl for latest kernel rpm
+url="https://dl.fedoraproject.org/pub/fedora/linux/updates/$releasever/Everything/x86_64/Packages/k"
+
+# get the latest kernel rpm
+content="$(curl "$url/")"
+latestrpm="$(echo "$content" | sed -e 's/<[^>]*>//g' | awk '{print $1}' | grep -E "kernel-core-[0-9]+.[0-9]+.[0-9]+-[0-9]+.fc29.x86_64.rpm")"
+latestver="$(echo "$latestrpm" | sed 's/kernel-core-//; s/\.rpm//')"
+
+if [ "x$latestrpm" != "x" ]; then
+    tmpdir=$(mktemp -d)
+    # download latest kernel rpm
+    wget -q -O "$tmpdir/$latestrpm" "$url/$latestrpm"
```

Comment:
rpm signature verification should be here. See [here](https://github.com/QubesOS/qubes-linux-dom0-updates/blob/master/Makefile) how to do it without importing the key into system rpm keyring

Reply:
Thank you for your feedback. I will add it soon.

Reply:
Should I look for the keys inside the builder-rpm?

Reply:
I'd include the key here.